src/components/Admin/Module/DalFieldStoreEncoding.vue

```diff
@@ -10,8 +10,17 @@
       <b-form-checkbox
         v-if="allowOmitStrategy"
         v-model="use"
+        :disabled="disabled"
       >
         {{ label }}
+
+        <b-badge
```

Comment:
No need for a badge in this component, since you know its disabled if checkbox is not checked